packages/remix/src/generators/setup-tailwind/files/tailwind.config.ts__tpl__

```diff
@@ -0,0 +1,9 @@
+import type {Config} from "tailwindcss";
+
+export default {
+  content: ["./app/**/*.{js,jsx,ts,tsx}"],
```

Comment:
issue: this is ok for standalone workspaces, but we need to also include content from projects the app depends on. You can reference the Ocean repo for how we have it set up for Cloud: https://github.com/nrwl/ocean/blob/main/apps/nxcloud-remix/tailwind.config.js